addons/mail/static/src/core/common/im_status_service.js

```diff
@@ -43,15 +41,31 @@ export const imStatusService = {
             }
         };
         bus_service.addEventListener("connect", () => updateBusPresence(), { once: true });
-        bus_service.subscribe("bus.bus/im_status_updated", async ({ partner_id, im_status }) => {
-            if (session.is_public || !partner_id || partner_id !== user.partnerId) {
-                return;
+        bus_service.subscribe(
+            "bus.bus/im_status_updated",
+            async ({ im_status, partner_id, guest_id }) => {
+                console.log("getting bus message bus.bus/im_status_updated");
+                const store = env.services["mail.store"];
```

Comment:
to check: I think store should be added to the service dependencies. there will be no need to check if it is available. I believe there were circular dependencies between store/im_status_service when it was defined in bus. It might not be the case anymore.